### PR TITLE
chore: update dependencies

### PR DESCRIPTION
**Pull request by capJavert**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -75,29 +75,29 @@
...
     "ioredis": "^5.4.1",
-    "isbot": "^5.1.9",
+    "isbot": "^5.1.12",
     "jsonexport": "^3.2.0",
     "jsonwebtoken": "^9.0.2",
     "lodash": "^4.17.21",
     "markdown-it": "^14.1.0",
     "markdown-to-txt": "^2.0.1",
-    "mercurius": "^13.3.3",
+    "mercurius": "^v14.1.0",
```
Comment (capJavert) [inline package.json:100]: Changelog says no major breaking changes, also fixes vulnerability

File: package.json [hunk 2]
```diff
@@ -75,29 +75,29 @@
...
     "ioredis": "^5.4.1",
-    "isbot": "^5.1.9",
+    "isbot": "^5.1.12",
     "jsonexport": "^3.2.0",
     "jsonwebtoken": "^9.0.2",
     "lodash": "^4.17.21",
     "markdown-it": "^14.1.0",
     "markdown-to-txt": "^2.0.1",
-    "mercurius": "^13.3.3",
+    "mercurius": "^14.1.0",
```
Comment (capJavert) [inline package.json:100]: changelog says no major breaking changes 🙏

File: package.json [hunk 3]
```diff
@@ -75,29 +75,29 @@
...
     "graphql-relay": "^0.10.1",
     "graphql-scalars": "^1.23.0",
     "graphql-subscriptions": "^2.0.0",
     "graphql-upload": "^15.0.1",
     "he": "^1.2.0",
-    "highlight.js": "^11.9.0",
+    "highlight.js": "^11.10.0",
     "ioredis": "^5.4.1",
-    "isbot": "^5.1.9",
+    "isbot": "^5.1.12",
```
Comment (rebelchris) [inline package.json:94]: Ah I thought they stopped supporting this for some reason 😅

File: package.json [hunk 4]
```diff
@@ -75,29 +75,29 @@
     "date-fns-tz": "^2.0.0",
     "deepmerge": "^4.3.1",
     "eventsource": "^2.0.2",
-    "fast-json-stringify": "^5.16.0",
-    "fastify": "^4.28.0",
+    "fast-json-stringify": "^6.0.0",
```
Comment (rebelchris) [inline package.json:78]: This also didn't switch anything important?

**Review by rebelchris — APPROVED** [review]
Just some sanity check

File: package.json (hunk 4 above)
Comment (capJavert) [reply package.json:78]: nope, just fastify update under the hood, we are on proper version

**Review by sshanzel — APPROVED** [review]
Nice updates 🚀

**Pull request merged**